### PR TITLE
[14.0] [mig] multi_step_wizard

### PR DESCRIPTION
**Pull request by schhatbar-initos**

### CODE REVIEW

**Review by dsolanki-initos — APPROVED** [review]
LGTM

**Comment by schhatbar-initos** [comment]
@tarteo Can you please review this PR ?

**Comment by schhatbar-initos** [comment]
@LoisRForgeFlow Can you please review this PR?
Thanks in Advance!

**Comment by schhatbar-initos** [comment]
@TDu Can you please review this PR?
Thanks in Advance!

**Comment by schhatbar-initos** [comment]
@etobella Can you please review this PR?
Thanks in Advance!

File: multi_step_wizard/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from . import models
+from .tests import multi_step_wizard_test
```
Comment (etobella) [inline multi_step_wizard/__init__.py]: Is not a good idea to load this model, please, use odoo_test_helper as it is done in
https://github.com/OCA/server-ux/blob/14.0/base_tier_validation/tests/common.py#L9
this way we can avoid this model in the installation.

**Review by etobella — CHANGES_REQUESTED** [review]
I know that this was not done by you, but I think now is a good moment to fix it. WDYT?

**Comment by schhatbar-initos** [comment]
@etobella I've made the changes as suggested. Can you please check is it fine enough with using odoo_test_helper?

File: multi_step_wizard/tests/test_multi_step_wizard.py [hunk 2]
```diff
@@ -1,20 +1,24 @@
...
 
-from odoo.tests.common import TransactionCase
+from odoo.tests import common
 
 
-class TestMultiStepWizard(TransactionCase):
-    def setUp(self):
-        super().setUp()
-        self.MultiStepWizard = self.env["multi.step.wizard.test"]
+class CommonTierValidation(common.SavepointCase):
```
Comment (simahawk) [inline multi_step_wizard/tests/test_multi_step_wizard.py]: `TestMultiStepWizard` and replace all calls to super w/ `super()`, no need for the class name.

**Review by simahawk — APPROVED** [review]
pre-approving

**Review by dsolanki-initos — APPROVED** [review]

**Comment by schhatbar-initos** [comment]
@simahawk Can you re-review this PR?
Thanks in Advance!

**Comment by simahawk** [comment]
@etobella fine for you?

**Review by etobella — APPROVED** [review]

**Comment by schhatbar-initos** [comment]
@simahawk Can you merge this PR hence it is approved?
Thanks in Advance!

**Pull request merged**